compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STMissingToken.java

```diff
@@ -25,27 +25,28 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
 /**
- * This is a generated internal syntax tree node.
+ * Represents a missing token in the internal syntax tree.
```

Comment:
Is this part of a Generated code?

Reply:
I doubled checked. It is not generated. That's why changed the description